src/types.jl

```diff
@@ -84,9 +84,7 @@ Base.one( s::Symbolic) = one( symtype(s))
 
 Base.zero(s::Symbolic) = zero(symtype(s))
 
-@noinline function promote_symtype(f, xs...)
-    error("promote_symtype($f, $(join(xs, ", "))) not defined")
-end
```

Comment:
Are you sure about this?

Reply:
I think so. Otherwise, if your symtype is `Any`, you get things like `promote_symtype(+, Any, Int)` not defined which is really annoying. 

Simply promoting to `Any` seems like a sensible option

Reply:
Okay makes sense!